### ISSUE TITLE
3.0.1 firmware on regular OnlyKey?

### ISSUE THREAD

**Issue by grivanov** [trustcrypto/OnlyKey-Firmware#139]
Is the 3.0.1 Firmware JUST for OnlyKey DUO or I can load it on the regular OnlyKey? Perhaps distinguish which firmware is for which OnlyKey in the release notes :)

**Comment by onlykey** [comment]
@grivanov Yes you can load this firmware on OnlyKey, but the only changes from v2.1.2 are for OnlyKey DUO support so there is really no reason to do this. Once we get a release that has new features for both DUO and OnlyKey we will publish an announcement.

**Comment by Zylanx** [comment]
I loaded it onto a normal OnlyKey, and it caused it to now act like a DUO, with only two buttons.
It also can't downgrade the firmware.
